src/main/java/net/minestom/server/world/biomes/BiomeManager.java

```diff
@@ -14,13 +16,20 @@
 /**
  * Allows servers to register custom dimensions. Also used during player joining to send the list of all existing dimensions.
  * <p>
- * Contains {@link Biome#PLAINS} by default but can be removed.
  */
 public final class BiomeManager {
     private final Map<Integer, Biome> biomes = new ConcurrentHashMap<>();
 
     public BiomeManager() {
-        addBiome(Biome.PLAINS);
+        // Need to register plains for the client to work properly
+        // Plains is always ID 0
+        addBiome(BiomeImpl.get("minecraft:plains"));
+    }
+
+    public void loadVanillaBiomes() {
+        for (BiomeImpl biome : BiomeImpl.values()) {
+            if (biome.id() != 0) addBiome(biome);
```

Comment:
Should `getByName` first to see if registered (or add a `has` method), and not load if so. rather than checking by ID.